src/top-bar/clear-all-button/clear-all-button.js

```diff
@@ -0,0 +1,17 @@
+import i18n from '@dhis2/d2-i18n'
+import { Button } from '@dhis2/ui'
+import React from 'react'
+import { useSelection } from '../selection/index.js'
+import classes from './clear-all-button.module.css'
+
+const ClearAllButton = () => {
+    const { clearAll, period, orgUnit } = useSelection()
+
+    return period.code || orgUnit.id ? (
```

Comment:
See comment regarding the reducer; If the reducer does not need the payload, then I'd assume that this check should also check whether there has been any workflow selected (also: I'd make a lot more sense to me to also clear the selected workflow when clicking on a button that says "Clear selections")